test/extended/deployments/deployments.go

```diff
@@ -263,7 +263,7 @@ var _ = g.Describe("deploymentconfigs", func() {
 
 			g.By("verifying the post deployment action happened: tag is set")
 			var out string
-			pollErr := wait.PollImmediate(100*time.Millisecond, 10*time.Second, func() (bool, error) {
+			pollErr := wait.PollImmediate(100*time.Millisecond, 1*time.Minute, func() (bool, error) {
```

Comment:
I'm aware of 5 mins here and there, but the less we have them the better for us. Those timeouts cost us longer build/test times in the queue. On the other hand flakes cost us repeated times, as well :/